### PR TITLE
ES-831441-AWS lambda

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Added Main page and Convert Word to PDF UG in  AWS Lambda

Regards,
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+title: Convert Word to PDF in Amazon Web Services on Lambda | Syncfusion
+description: Convert Word document to PDF in  Amazon Web Services on Lambda using .NET Core Word (DocIO) and  PDF library, without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word Document to PDF in Amazon Web Services on Lambda
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in Amazon Web Services on Lambda**
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:11]: Remove unwanted spaces and comma.
Add full stop.

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+## Steps to convert Word document to PDF in Amazon Web Services on Lambda
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:13]: Bold the options as per KB.
Remove unwanted empty lines.

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<thead>
+<tr>
+<th>
+Amazon Web Services<br/></th><th>
+Nuget package name<br/></th></tr></thead>
+<tr>
+<td>
+Lambda <br/></td><td>
+{{'[Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core)' | markdownify}}<br/>
+{{'[SkiaSharp.NativeAssets.Linux v2.8.2.3](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux)' | markdownify}} <br/></td></tr>
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md:31]: Also mention [HarfBuzzSharp.NativeAssets.Linux v2.8.2.2](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux/2.8.2.2) NuGet

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to PDF in Amazon Web Services** within a few lines of code. 
+
+N> If this is your first time working with Amazon Web Services, please refer to the dedicated Amazon Web Services resources. This section explains how to convert Word documents to PDF in C# using the .NET Core Word (DocIO) library in Amazon Web Services (AWS). 
+
+## Prerequisites 
+
+* An active **Amazon Web Services account** is required. If you don’t have one, please create a account before starting.
+
+## Nuget package required
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md:19]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md:31]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:13]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to PDF in Amazon Web Services** within a few lines of code. 
+
+N> If this is your first time working with Amazon Web Services, please refer to the dedicated Amazon Web Services resources. This section explains how to convert Word documents to PDF in C# using the .NET Core Word (DocIO) library in Amazon Web Services (AWS). 
+
+## Prerequisites 
+
+* An active **Amazon Web Services account** is required. If you don’t have one, please create a account before starting.
+
+## Amazon Web Services
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: ## Amazon Web Services (AWS)

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+---
+title: Convert Word to PDF in Amazon Web Services | Syncfusion
+description: Learn how to convert a Word document to a PDF in Amazon Web Services using Syncfusion .NET Core Word (DocIO) library in C#.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to PDF in Amazon Web Services
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: Amazon Web Services (AWS)

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+---
+title: Convert Word to PDF in Amazon Web Services | Syncfusion
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: Convert Word to PDF in AWS | Syncfusion

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+# Convert Word to PDF in Amazon Web Services 
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to PDF in Amazon Web Services** within a few lines of code. 
+
+N> If this is your first time working with Amazon Web Services, please refer to the dedicated Amazon Web Services resources. This section explains how to convert Word documents to PDF in C# using the .NET Core Word (DocIO) library in Amazon Web Services (AWS). 
+
+## Prerequisites 
+
+* An active **Amazon Web Services account** is required. If you don’t have one, please create a account before starting.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: https://aws.amazon.com/

Download and install the AWS Toolkit for Visual Studio, you can download the AWS toolkit from this [link](https://aws.amazon.com/visualstudio/). The Toolkit can be installed from Tools/Extension and updates options in Visual Studio.

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+writer.Close();
+System.Diagnostics.Process.Start("Sample.pdf");
+
+{% endhighlight %}
+
+{% endtabs %}
+
+By executing the program, you will get the **PDF document** as follows.
+
+![Install SkiaSharp.NativeAssets.Linux Nuget Package](WordToPDF_images/WordToPDF_Output_Cloud.png)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:169]: Check the alt text and add proper

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.Pdf;
+using Syncfusion.Drawing;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:45]: Remove this

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+ 
+    WordDocument document = new WordDocument(fileStream, FormatType.Docx); 
+    DocIORenderer render = new DocIORenderer();
+    PdfDocument pdf = render.ConvertToPDF(document);
+ 
+    //Save the document into stream
+    MemoryStream stream = new MemoryStream();
+
+    //Save the PDF document  
+    pdf.Save(stream);
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:78]: pdf.Close()

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+    document.Close();
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:79]: //Releases all resources used by the Word document and DocIO Renderer objects

//Closes the PDF document

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md (hunk 8 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: UPDATED

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md (hunk 7 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Amazon-Web-Service.md]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md (hunk 10 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:45]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md (hunk 11 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:78]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md (hunk 12 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS-Lambda.md:79]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+# Convert Word to PDF in Amazon Web Services (AWS)
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to PDF in Amazon Web Services (AWS)** within a few lines of code. 
+
+N> If this is your first time working with Amazon Web Services (AWS), please refer to the dedicated AWS resources. This section explains how to convert Word documents to PDF in C# using the .NET Core Word (DocIO) library in AWS. 
+
+## Prerequisites 
+
+* An active **Amazon Web Services (AWS) account** is required. If you don’t have one, please [create a account](https://aws.amazon.com/) before starting.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS.md:17]: An active Amazon Web Services (AWS) account is required. If you don’t have one, please [create an account](https://aws.amazon.com/) before starting.

Address the same grammar "an account" for Azure home page also

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS.md (hunk 13 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS.md]: Updated

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**